app/views/core_induction_programme/lessons/show.html.erb

```diff
@@ -1,3 +1,7 @@
+<%= govuk_breadcrumbs breadcrumbs: [
+  ["Home", @current_user ? dashboard_path : cip_path],
+  [@course_lesson.course_module.course_year.title, cip_year_path(@course_lesson.course_module.course_year)],
+  [@course_lesson.course_module.title, cip_year_module_path(@course_lesson.course_module.course_year, @course_lesson.course_module)]] %>
```

Comment:
This is feeling a bit duplicatey, is there any way to clean this up easily?

Reply:
Eh, I guess we could have a helper for it. Will do :)